data-es2016plus.js

```diff
@@ -152,7 +142,6 @@ exports.tests = [
       babel: true,
       es7shim: true,
       typescript: typescript.corejs,
-      firefox45: firefox.nightly,
```

Comment:
I believe the correct here would be to change firefox45 to "flag", since it is what is implemented.